### PR TITLE
Catalystcoop.pudl

### PR DESCRIPTION
**Pull request by zaneselvans**
This is the first time I've submitted a package to conda-forge. I haven't yet been able to get docker up and running on my local machine, so I wasn't able to run the tests locally. Hopefully it all works! I wasn't totally clear on how the URL for the tarball is constructed, but I just left it as it was in the example.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by zaneselvans** [comment]
I confirm that I'm willing to be the maintainer for this package.

**Comment by zaneselvans** [comment]
Hmm, it's failing on `import pkg_resources` but that is [distributed as part of setuptools](https://setuptools.readthedocs.io/en/latest/pkg_resources.html) which is being installed, so I would have thought it would be available.

File: recipes/catalystcoop.pudl/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
```
Comment (isuruf) [inline recipes/catalystcoop.pudl/LICENSE.txt]: Remove this file as conda-build can look for this in the source tarball

File: recipes/catalystcoop.pudl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    - pudl_data --help
+    - ferc1_to_sqlite --help
+    - pudl_etl --help
+    - epacems_to_parquet --help
+
+about:
+  home: https://catalyst.coop/pudl
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```
Comment (isuruf) [inline recipes/catalystcoop.pudl/meta.yaml]:
```suggestion
  license_file: LICENSE
```

File: recipes/catalystcoop.pudl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{% set name = "catalystcoop.pudl" %}
+{% set version = "0.1.0a2" %}
```
Comment (isuruf) [inline recipes/catalystcoop.pudl/meta.yaml]: Why package an alpha release where there's 0.1.3?

Comment (zaneselvans) [reply recipes/catalystcoop.pudl/meta.yaml]: I'm guessing you're referring to [this package](https://pypi.org/project/pudl/), which sadly isn't ours. We have reached out to the developers, and they want to retain that name, even though the package hasn't been updated in a couple of years. We've been working on `catalystcoop.pudl` for about three years, and have users that have integrated it into their code code, and we have several thousand lines of our own, so using the `organization.package` format seemed easiest.

Comment (zaneselvans) [reply recipes/catalystcoop.pudl/meta.yaml]: More generally, as to why a pre-release -- we're trying to get the packaging and distribution details worked out in parallel with some final code changes in advance of a workshop next week where we want folks to be able to install the package easily, but the docs don't match up with the code right now, and we don't want to confuse people by pushing it out while it's still known to be incongruous.

Comment (zaneselvans) [reply recipes/catalystcoop.pudl/meta.yaml]: @isuruf Do you feel like the naming/versioning here is a blocking issue?

**Comment by zaneselvans** [comment]
Ah, Windows console UTF-8 character encoding.

**Pull request merged**